src/js/firstDiscoveryEditor.js

```diff
@@ -107,6 +107,18 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                     messageBase: "{messageLoader}.resources.prefsEditor.resourceText"
                 }
             },
+            stepCount: {
```

Comment:
What do you think about consolidating all of these nav related components into a nav component? I suppose the new nav component could still be a collection of these sub components in some way.